agrirouter-api-java-api/src/main/kotlin/com/dke/data/agrirouter/api/service/parameters/MessageConfirmationParameters.kt

```diff
@@ -14,15 +14,6 @@ class MessageConfirmationParameters : ParameterValidation {
 
     @NotNull
     lateinit var onboardingResponse: OnboardingResponse
-
```

Comment:
Are those changes part of the current PR? Mixing different functional features within one PR is a PITA.

(ง'̀-'́)ง

Reply:
Same here, good eyes :cop: